nntrainer/layers/flatten_layer.cpp

```diff
@@ -41,21 +41,14 @@ int FlattenLayer::initialize() {
   return status;
 }
 
-sharedConstTensors FlattenLayer::forwarding(sharedConstTensors in) {
-  input = *in[0];
-  hidden = input;
-
-  hidden.reshape(output_dim[0]);
-
-  return {MAKE_SHARED_TENSOR(hidden)};
+void FlattenLayer::forwarding(sharedConstTensors in) {
+  memcpy(net_hidden[0]->var.getData(), net_input[0]->var.getData(),
```

Comment:
is it possible to set net_hidden[0] = net_input[0] here?

Reply:
If we use like this, the dimension is also copyed. So I don't think it is possible.

Reply:
yes, i meant like below:
```
Tensor temp = net_input[0];
temp.setDim(net_hidden[0].getDim());
net_hidden[0] = temp;
```

Reply:
done.